### PR TITLE
Secondaries

### PR DESCRIPTION
**Pull request by aaishaa**

### CODE REVIEW

File: wp-content/themes/lundy-law/template-attorney-overview.php [hunk 1]
```diff
@@ -44,12 +40,33 @@
...
 					
-					<div class="attorney-listing">
-						<a href="<?php echo get_permalink(); ?>"><img alt="attorney photo" src="<?php echo $attorney_photo ;?>" /></a>
-						<br />
-						<a href="<?php echo get_permalink(); ?> "><?php echo $attorney_name ;?></a>
+					<div class="attorney-listing white-text element-item">
+						<a href="<?php echo get_permalink(); ?>">
+							<div class="attorney-quote">
+								<img class="attorney-photo" src="<?php echo $attorney_photo ;?>" />
+								<blockquote>
```
Comment (matt-cos) [inline wp-content/themes/lundy-law/template-attorney-overview.php]: @aaishaa the blockquotes sit too low for some of these attorneys. if you vertically center everyone's quote, it should take care of that. also, the quotation marks should be a little bigger

File: wp-content/themes/lundy-law/development/styles/style.scss [hunk 2]
```diff
@@ -326,6 +328,9 @@ body.page-template {
 		-moz-appearance: none;
 		text-transform: uppercase;
 	}
+	select.language option:after{
```
Comment (matt-cos) [inline wp-content/themes/lundy-law/development/styles/style.scss]: @aaishaa lol i think we can delete this

File: wp-content/themes/lundy-law/template-testimonials.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+	the_post(); ?>
+
+		<?php get_template_part( 'partials/page_title' ); ?>
+		<div class="testimonials banner clearfix">
+			<h1>Hear What Customers<br/>Are Saying</h1>
+		</div>
+		<div class="container clearfix testimonials">
+			<div class="content">
+				<div class="shell">
+					<div class="videos">
```
Comment (aaishaa) [inline wp-content/themes/lundy-law/template-testimonials.php]: Need more context for this: Which line needs this class? What part of the tracker does this apply to? And will adding the code above conflict with existing code in _grid.scss on line 131? : `.twelve.xs-no-padding,
.xs-twelve.xs-no-padding {
	width: $column * 12;
	padding: 0;
}`

File: wp-content/themes/lundy-law/development/styles/style.scss (hunk 2 above)
Comment (aaishaa) [inline wp-content/themes/lundy-law/development/styles/style.scss]: lol—agreed. Pushed changes.

File: wp-content/themes/lundy-law/template-testimonials.php [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+					<!-- end of twelve columns -->
+				</div>
+				<!-- end of shell -->
+			</div>
+			<!-- end of content -->
+
+		</div>
+		<!-- end of container -->
+		<script>
+			jQuery(document).ready(function($){
```
Comment (aaishaa) [inline wp-content/themes/lundy-law/template-testimonials.php]: Deleted!

File: wp-content/themes/lundy-law/template-testimonials.php (hunk 3 above)
Comment (aaishaa) [inline wp-content/themes/lundy-law/template-testimonials.php]: Added the changes anyway (below line 131)—they worked fine.

File: wp-content/themes/lundy-law/development/styles/style.scss [hunk 5]
```diff
@@ -550,6 +555,77 @@ body.page-template {
...
+		background-image: url(../../uploads/2014/04/subpage-banner.jpg);
+		background-size: cover;
+		background-position: center;
+		width: 100%;
+		color: #fff;
+	}
+	.videos{
+		// width:90%;
+		margin: 0 auto;
+		display: flex;
```
Comment (aaishaa) [inline wp-content/themes/lundy-law/development/styles/style.scss]: @matt-cos removing the flexbox brings back the alignment issue noted above. Padding needs to be distributed between the elements in order to keep the video columns aligned with the blockquotes

File: wp-content/themes/lundy-law/template-attorney-overview.php (hunk 1 above)
Comment (aaishaa) [inline wp-content/themes/lundy-law/template-attorney-overview.php]: @matt-cos thanks. Pushed a fix to make the blockquotes vertically aligned, and made the quotes larger.

**Pull request merged**